src/bin/cql-stress-scylla-bench/args.rs

```diff
@@ -332,6 +339,9 @@ where
         // therefore just subtract with wraparound and treat u64::MAX as infinity
         let max_consecutive_errors_per_op = max_errors_at_row.get().wrapping_sub(1);
 
+        // Similar to above
+        let max_errors_in_total = max_errors.get().wrapping_sub(1);
```

Comment:
Just to make sure i understand correctly - this option (unlike `error-at-row-limit`) defines the number of errors after which the whole run fails. In other words, when `error-limit = 1`, we immediately fail the run after the first error occurs. Am i right?

If so, then this `wrapping_sub(1)` looks semantically correct to me (looking at the `WorkerContext::decrement_global_retry_counter`).